new/prusti-specs/src/specifications/untyped.rs

```diff
@@ -147,6 +153,11 @@ impl EncodeTypeCheck for Assertion {
             AssertionKind::Expr(expression) => {
                 expression.encode_type_check(tokens);
             }
+            AssertionKind::And(vec_assertions) => {
```

Comment:
It would be more consistent with the rest of Prusti to use `assertions` instead of `vec_assertions`.